From 0b80048e91f5fd48429c9264d8fd86c4f273ba6e Mon Sep 17 00:00:00 2001
From: Anthoine Bourgeois <anthoine.bourgeois@vates.tech>
Date: Sat, 27 Sep 2025 00:10:13 +0200
Subject: [PATCH] qcow2_helper: Fix bound array warning

This fix a warning about a mismatch between
QCOW2_HEADER_SIZE and the size of qcow2_header structure.

Signed-off-by: Anthoine Bourgeois <anthoine.bourgeois@vates.tech>
---
 qcow2/qcow_helper.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/qcow2/qcow_helper.h b/qcow2/qcow_helper.h
index 2d241489..9d9725ec 100644
--- a/qcow2/qcow_helper.h
+++ b/qcow2/qcow_helper.h
@@ -37,12 +37,6 @@ struct qcow2_header {
 
     uint32_t refcount_order;
     uint32_t header_length;
-
-    /* Additional fields */
-    uint8_t compression_type;
-
-    /* header must be a multiple of 8 */
-    uint8_t padding[7];
 } __attribute__((packed));
 
 #define SWAP_BE_TO_LE(size, x) \
